Sources/KlaviyoSwift/KlaviyoAPI.swift

```diff
@@ -33,31 +33,27 @@ struct KlaviyoAPI {
     }
 
     // For internal testing use only
-    static var requestStarted: (KlaviyoRequest) -> Void = { _ in }
-    static var requestCompleted: (KlaviyoRequest, Data, Double) -> Void = { _, _, _ in }
-    static var requestFailed: (KlaviyoRequest, Error, Double) -> Void = { _, _, _ in }
-    static var requestRateLimited: (KlaviyoRequest, Int?) -> Void = { _, _ in }
-    static var requestHttpError: (KlaviyoRequest, Int, Double) -> Void = { _, _, _ in }
-
+    static var requestCompletion: (KlaviyoRequest, URLRequest?, RequestStatus) -> Void = { _, _, _ in }
```

Comment:
Nice this is a lot simpler. 

Nit pick comment - Would it make sense to rename the closure to something `requestHandler` or something? completion seems to me like this closure is called when the request completes?

Reply:
yeah good call, I'll change it